build.gradle.kts

```diff
@@ -21,6 +22,7 @@ buildscript {
     dependencies {
         classpath(GradlePlugins.ANDROID)
         classpath(kotlin(module = GradlePlugins.Kotlin.ID, version = GradlePlugins.Kotlin.VERSION))
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
```

Comment:
:mag: I think this is not correct? @pavlospt

Reply:
🤔🤔🤔🤔

Reply:
@charbgr fixed!

Reply:
![86225012-5130e680-bb92-11ea-8631-d8622645d97f](https://user-images.githubusercontent.com/3369653/87676141-e92cf380-c780-11ea-9bce-f965c6c978c1.gif)